pkg/util/hlc/hlc.go

```diff
@@ -204,10 +204,10 @@ func (c *Clock) Update(rt Timestamp) Timestamp {
 	}
 
 	// In the remaining cases, our physical clock plays no role
-	// as it is behind the local and remote wall times. Instead,
+	// as it is behind the local or remote wall times. Instead,
 	// the logical clock comes into play.
 	if rt.WallTime > c.mu.timestamp.WallTime {
-		offset := time.Duration(rt.WallTime-physicalClock) * time.Nanosecond
```

Comment:
I disagree with this change: the multiplication, although technically unneeded, serves as a reminder of the unit of time to the reader of this code. It is thus important for maintainability.